modules/crud.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete a single todo
+export const deleteTodo = (id, itemContainer, todoListss) => {
+  const newTodos = todoListss.filter((item) => item.id !== id);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+  if (itemContainer.parentNode) {
+    itemContainer.parentNode.removeChild(itemContainer);
+  }
+};
```

Comment:
- [ ] [OPTIONAL] This requirement is optional because the previous reviewer skipped it, however I urge you to implement it. As per requirement, deleting a task should update all remaining items' indexes so they represent the current list order and are unique. Currently, deleting a task from your to-do list doesn't re-index the remaining tasks.
![image](https://user-images.githubusercontent.com/46839548/214776239-e14e2b87-36a6-4947-91f3-428656752163.png)